rapid_pro_tools/rapid_pro_client.py

```diff
@@ -143,9 +143,53 @@ def get_flow_definitions_for_flow_ids(self, flow_ids):
         :rtype: temba_client.v2.types.Export
         """
         return self.rapid_pro.get_definitions(flows=flow_ids, dependencies="all")
+    
+    def _get_archived_messages(self, created_after_inclusive=None,
+                                       created_before_exclusive=None):
+        """
+        Gets the raw messages from Rapid Pro's archives.
+        
+        Uses the created dates to determine which archives to download.
+
+        :param created_after_inclusive: Start of the date-range to download messages from.
+                                              If set, only downloads messages created since that date,
```

Comment:
```suggestion
                                        If set, only downloads messages created since that date,
```
and similar for the other over-indented lines in this docstring